### PR TITLE
More consistency with client vs device terminology

### PR DESCRIPTION
**Pull request by alavers**

### CODE REVIEW

File: slate/source/includes/rest/_core.md [hunk 1]
```diff
@@ -757,13 +757,13 @@ smooch.appUsers.init({
...
+This API is called by an iOS, Android, or browser client when the app is first loaded. It serves a number of purposes:
 
-1. Initializes a new `appUser` and `device` if they don't yet exist
-1. Updates an existing app user's profile and device information
+1. Initializes a new `appUser` and `client` if they don't yet exist
+1. Updates an existing app user's profile and client information
 1. Authenticates the `appUser` if `jwt` credentials are provided
 
-The API requires that a `device` be specified at the very minimum. A `userId` may also be specified to [link app user accounts across devices](/#users-on-multiple-devices).
+The API requires that a `device` parameter be specified at the very minimum. A `userId` may also be specified to [link app user accounts across multiple clients](/guide/multi-client-users).
```
Comment (jugarrit) [inline slate/source/includes/rest/_core.md]: `client`?

File: slate/source/includes/rest/_core.md [hunk 2]
```diff
@@ -757,13 +757,13 @@ smooch.appUsers.init({
...
 
 The API responds with an `appUser` object. The `appUser` includes an `_id` that can be used to make further API calls on behalf of that user. If the `userId` and/or `device.id` are seen for the first time a new `appUser` will be created. If not, the existing `appUser` be returned.
```
Comment (jugarrit) [inline slate/source/includes/rest/_core.md]: `client.id`?

File: slate/source/includes/rest/_core.md (hunk 1 above)
Comment (jugarrit) [inline slate/source/includes/rest/_core.md]: Ehh never mind, our API is `device` only...

File: slate/source/includes/rest/_core.md (hunk 2 above)
Comment (jugarrit) [inline slate/source/includes/rest/_core.md]: Ehh never mind, our API is device only...

File: slate/source/includes/rest/_core.md [hunk 3]
```diff
@@ -757,13 +757,13 @@ smooch.appUsers.init({
 
 <api>`POST /v1/init`</api>
```
Comment (jugarrit) [inline slate/source/includes/rest/_core.md]: I find this confusing now that it says client everywhere except for the init API.

**Comment by jpjoyal** [comment]
Changes here are great to be more consistent. But I'd really like to see a more holistic doc on Account linking/user merging/user id usage. We don't seem to have a page that covers everything about multi channel users

File: pages/guide/multi-client-users.md [hunk 4]
```diff
@@ -1,18 +1,18 @@
...
+If you want the user to see the same conversation across clients you can do so by assigning your users a `userId`. This will link a user across all clients and platforms that have integrated with your Smooch app. Specifying a `userId` also facilitates [user authentication](/guide/authenticating-users/).
 
-The `userId` is a string that can be an email, GUID, or an existing ID from your own user directory. The `userId` must be unique within a given Smooch app, and it will only link devices within a single Smooch app.
+The `userId` is a string that can be an email, GUID, or an existing ID from your own user directory. The `userId` must be unique within a given Smooch app, and it will only link clients within a single Smooch app.
 
 <aside class="warning">
 **Caution:** If you're specifying a `userId` then in order to keep conversations private we strongly suggest [authenticating your users](/guide/authenticating-users/). If a `userId` is used without a JWT credential, then anyone who can discover a user's `userId` could potentially eavesdrop on the conversation.
```
Comment (jpjoyal) [inline pages/guide/multi-client-users.md]: authenticating-users link is broken.

Comment (dannytranlx) [inline pages/guide/multi-client-users.md]: looks like markdown is broken instead

Comment (alavers) [inline pages/guide/multi-client-users.md]: Yeah. I'm simply going to remove the aside tag for now.

Comment (lemieux) [inline pages/guide/multi-client-users.md]: Just use a `strong` tag?

Comment (alavers) [inline pages/guide/multi-client-users.md]: Well, the word Caution is effectively `strong`ed now

**Pull request merged**